### ISSUE TITLE
Multiple stacks

### ISSUE THREAD

**Issue by MateuszSiek** [karelnagel/pulumi-remotion-lambda#1]
Thanks for creating this script, makes the deployment with sst so much easier!

I'm now trying to make it work with multiple environments which seem quite challenging after a bit of investigation. 
I was hoping I can simply make various names relate to the env name but according to [documentation](https://www.remotion.dev/docs/lambda/setup) the Policy and Role should have an **exact** name. 
Trying to use the same names in different environments with sst/pulumi throws errors about existing resources.

To add, my use case is that I need to have different deployments of the remotion composer for different environments. That said I should probably be fine with using the same function, roles etc, so probably there is no need to have multiple deployment of those. 
For the bucket I could either have various buckets or have various deployments within the same bucket(which seem like an easier option).
But still, trying to reuse the functions etc across sst deployments makes the pulumi throw errors about existing resources.

Do you have some idea how the multi env could be achieved?

**Comment by karelnagel** [comment]
Thanks! 
Yes the multiple stacks possiblity would be very necessary for me as well, but since I haven't used this stack in a real project yet, I haven't had the need for this so far. I tried just with having different names for roles/policy/etc, but that attempt failed.

Since you mention that same role, policy, function would be ok to have for multiple stacks and only deploy the site for each env, maybe this would be easier to implement. Just by checking if the role/policy/function exists and not creating new if there is one already. But yeah since this is my second attempt to make something with Pulumi, I would have to search more to figure out how to do it. 

TLDR: It can be done, but I don't have time/interest to do it rn, maybe someday, but PRs are welcome 😀

**Comment by karelnagel** [comment]
trying to solve this here: https://github.com/karelnagel/pulumi-remotion-lambda/pull/3

**Comment by karelnagel** [comment]
Well that was easier than I thought 😀

@MateuszSiek you can try the new 0.0.5 version if it works for you as well.

**Comment by MateuszSiek** [comment]
Ohh, that sounds great, will try to take a look in the upcoming days. Thanks!
